inc/load.php

```diff
@@ -118,6 +118,16 @@ function load_autoload($name){
         return true;
     }
 
+    // plugin namespace
+    if(substr($name, 0, 7) == 'plugin/') {
+        $name = str_replace('/test/', '/_test/', $name); // no underscore in test namespace
```

Comment:
With my note about the subfolder I mean:
`use dokuwiki\plugin\myplugin\test\something\test\thing` is converted into: `plugin/myplugin/_test/something/_test/thing`

I expect that a test subfolder will not be used regularly, but it is therefore a design choice at least.